deploy/registration-service.yaml

```diff
@@ -172,8 +221,16 @@ objects:
     data:
       environment: ${ENVIRONMENT}
       auth_client.library_url: ${AUTH_CLIENT_LIBRARY_URL}
-      auth_client.config_raw: ${AUTH_CLIENT_CONFIG_RAW}
       auth_client.public_keys_url: ${AUTH_CLIENT_PUBLIC_KEYS_URL}
+      auth_client.config_raw.realm: ${AUTH_CLIENT_CONFIG_RAW_REALM}
```

Comment:
Why are you spiting the raw configuration? I would still keep the raw configuration as a string. The reg-service doesn't really care what this string is about. It's just a string it needs to pass to the UI client.

Reply:
It just seemed cleaner to me, it means we don't need to decode the value if we need to access a field (this is done in the tests) and it also makes it easier when creating a configmap imo. lastly, it seems a bit inconsistent with the other fields in the dev.yaml file and now the configmap as well. in any case, i've changed it back